test/test.js

```diff
@@ -1,27 +1,50 @@
 function readBlobAsText(blob) {
   return new Promise(function(resolve, reject) {
```

Comment:
This would make the return value of the ``readBlobAsText`` function inconsistent. Other parts of the library are expecting to get a promise from it